src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardRetryingRequestResolver.java

```diff
@@ -238,12 +235,12 @@ public void updateData(@NotNull final Map<IToken, Integer> newAssignedRequests,
         this.delays.putAll(newDelays);
     }
 
-    public HashMap<IToken, Integer> getDelays()
+    public HashMap<IToken<?>, Integer> getDelays()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "Map" rather than the implementation "HashMap". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1319)